### PR TITLE
Erisports

### PR DESCRIPTION
**Pull request by cdb-is-not-good**
## About The Pull Request
<details>
<summary>
Ports: https://github.com/https://github.com/discordia-space/CEV-Eris/pull/8369
fix: Robotics Bioprinter now prints needed 2 different kidneys instead of 2-in-1
fix: Brain now can acquire "sharp" injuries
fix: Weapons that have twohanded sprites now are properly drawn in right hand.
fix: All scanners (including medical scanner) now have power consumption per use.
fix: Welding without welding goggles damages eyes.
fix: You can stab someone's eyes with screwdrivers again, the sound is present too.
tweak: Alkysine and Imidazoline now heal Brain and Eyes without hacks
fix: Oxyrush, Trauma Control System, Nanosymbiotes now function properly.
balance: Trauma Control System and Nanosymbiotes exchanged recipes.
add: Added new brain and eyes wounds.
fix: Cyanide should kill you now.
fix: Surgery slips should damage properly.
</summary>
<hr>

	
<hr>
</details>

## Changelog
:cl:

/:cl:

### CODE REVIEW

**Comment by cdb-is-not-good** [comment]
There we go, no more accidental commits added.

File: maps/__Nadezhda/map/Underground_Old.dmm [hunk 1]
```diff
@@ -59874,7 +59874,7 @@
 /area/nadezhda/quartermaster/miningdock)
 "cRG" = (
 /obj/structure/table/reinforced,
-/obj/item/stack/medical/advanced/bruise_pack,
```
Comment (Trilbyspaceclone) [inline maps/__Nadezhda/map/Underground_Old.dmm:59877]: wrong pathing

File: code/modules/organs/internal/internal_wounds/brain.dm [hunk 2]
```diff
@@ -0,0 +1,236 @@
+
+// Blunt
+/datum/component/internal_wound/organic/brain_blunt
+	treatments_item = list(/obj/item/stack/medical/bruise_pack/advanced = 2)
```
Comment (Trilbyspaceclone) [inline code/modules/organs/internal/internal_wounds/brain.dm:4]: every case of this is wrong pathing

File: code/modules/organs/internal/internal_wounds/eyes.dm [hunk 3]
```diff
@@ -0,0 +1,244 @@
+// Blunt
+/datum/component/internal_wound/organic/eyes_blunt
+	treatments_item = list(/obj/item/stack/medical/bruise_pack/advanced = 2)
```
Comment (Trilbyspaceclone) [inline code/modules/organs/internal/internal_wounds/eyes.dm:3]: same in this file

File: code/game/objects/items/devices/scanners/health.dm [hunk 4]
```diff
@@ -2,7 +2,7 @@
 /obj/item/device/scanner/health
 	name = "health analyzer"
 	desc = "A hand-held body scanner able to distinguish vital signs of the subject."
-	icon_state = "health"
+	icon_state = "health0"
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/devices/scanners/health.dm:5]: we dont have health0, its just health

File: code/game/objects/items/devices/scanners/health.dm [hunk 5]
```diff
@@ -253,3 +253,21 @@
...
 	. = jointext(dat, "<br>")
+
+/obj/item/device/scanner/health/cell_use_check(charge, mob/user)
+	. = TRUE
+	icon_state = "health"
+	if(!cell || !cell.checked_use(charge))
+		if(user)
+			to_chat(user, SPAN_WARNING("[src] battery is dead or missing."))
+		. = FALSE
+		icon_state = "health0"
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/devices/scanners/health.dm:264]: Same as above

File: code/modules/reagents/reagents/medicine.dm [hunk 6]
```diff
@@ -635,7 +629,7 @@ We don't use this but we might find use for it. Porting it since it was updated
 		if(E && istype(E))
 			var/list/current_wounds = E.GetComponents(/datum/component/internal_wound)
 			if(LAZYLEN(current_wounds) && prob(10))
-				LEGACY_SEND_SIGNAL(E, COMSIG_IORGAN_REMOVE_WOUND, pick(current_wounds))
+				SEND_SIGNAL(E, COMSIG_IORGAN_REMOVE_WOUND, pick(current_wounds))
```
Comment (Trilbyspaceclone) [inline code/modules/reagents/reagents/medicine.dm:632]: these should be LEGACY_SEND_SIGNAL

**Comment by cdb-is-not-good** [comment]
forgot to remark this. oops

**Pull request merged**